src/Graph.elm

```diff
@@ -1217,12 +1175,13 @@ heightLevels (AcyclicGraph graph _) =
             in
             case IntDict.get id indegreesDec of
                 Just 0 ->
-                    case get id graph of
+                    case get id startingGraph of
                         Just ctx ->
                             ( ctx :: nextLevel, indegreesDec )
 
                         Nothing ->
-                            Debug.crash "Graph.heightLevels: Could not get a node of a graph which should be there by invariants. Please file a bug report!"
+                            -- Debug.crash "Graph.heightLevels: Could not get a node of a graph which should be there by invariants. Please file a bug report!"
+                            ( nextLevel, indegreesDec )
```

Comment:
This will silently fail.

Reply:
dito. Please call the `crashLoop` with appropriate `reason`, so that people can figure out why it crashed when looking at it in the debugger.